services/frontend/src/components/errorResult/errorProfilePage.jsx

```diff
@@ -6,7 +6,7 @@ import { HomeOutlined, UserOutlined } from "@ant-design/icons";
 import { useSelector } from "react-redux";
 import ErrorResultView from "./errorResultView";
 
-const ErrorProfileHidden = () => {
+const ErrorProfilePage = (title, subtitle) => {
```

Comment:
Same situation as above, and add PropTypes. Since the "title" and "subtitle" prop is technically not the title/subtitle, can you rename it to something related to the formattedMessage, maybe titleId and subtitleId?

```suggestion
const ErrorProfilePage = ({ title, subtitle }) => {
```